uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/BaseEditor.java

```diff
@@ -581,8 +599,6 @@ private void disableMenuItem(final MenuItems menuItem) {
         }
     }
 
-    protected boolean isValidationRunning = false;
```

Comment:
IIRC this was used to prevent the User from invoking "validation" twice in succession. 

Is this flag no longer used? @Rikkola may remember more.

Reply:
@manstis You scared me! It is still there, just moved few lines up. Looks like I left it into a stupid location.

Reply:
You scared me! [2] 😰 It was just the auto code format.